apps/analog-demo-auth/.env.example

```
@@ -0,0 +1,10 @@
+# OAuth/OIDC Provider
+AUTH_ISSUER=http://localhost:8080/realms/dev
+AUTH_CLIENT_ID=local
+AUTH_CLIENT_SECRET=QAZKVBwuFROLVu9u64HwOfy65SR1Q7dD
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_CLIENT_SECRET key should go before the AUTH_ISSUER key

(UnorderedKey)